### PR TITLE
add tejolote prow jobs

### PR DESCRIPTION
**Pull request by cpanato**
- add tejolote prow jobs

needed for https://github.com/kubernetes-sigs/tejolote/pull/3

/assign @puerco @xmudrii @Verolop @ameukam @saschagrunert 
cc @kubernetes/release-engineering

### CODE REVIEW

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

File: config/jobs/kubernetes-sigs/tejolote/tejolote-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+presubmits:
+  kubernetes-sigs/tejolote:
+  - name: pull-tejolote-test
```
Comment (ameukam) [inline config/jobs/kubernetes-sigs/tejolote/tejolote-presubmits.yaml:3]:
```suggestion
  - name: pull-tejolote-test
    cluster: k8s-infra-prow-build
```

File: config/jobs/kubernetes-sigs/tejolote/tejolote-presubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        args:
+        - run
+        - mage.go
+        - Test
+    annotations:
+      testgrid-dashboards: sig-release-releng-presubmits
+      testgrid-tab-name: tejolote-test
+      testgrid-alert-email: release-managers+alerts@kubernetes.io
+      testgrid-num-columns-recent: '30'
+  - name: pull-tejolote-verify
```
Comment (ameukam) [inline config/jobs/kubernetes-sigs/tejolote/tejolote-presubmits.yaml:22]:
```suggestion
  - name: pull-tejolote-verify
    cluster: k8s-infra-prow-build
```

Comment (cpanato) [reply config/jobs/kubernetes-sigs/tejolote/tejolote-presubmits.yaml:23]: done, thanks

File: config/jobs/kubernetes-sigs/tejolote/tejolote-presubmits.yaml (hunk 1 above)
Comment (cpanato) [reply config/jobs/kubernetes-sigs/tejolote/tejolote-presubmits.yaml:3]: @ameukam should we update the other jobs for the other repos to use that as well?

Comment (ameukam) [reply config/jobs/kubernetes-sigs/tejolote/tejolote-presubmits.yaml:3]: Yeah. we should do it for all the other jobs. rule of the thumb is to have prowjobs running on community-owned insfrastructure.

Comment (cpanato) [reply config/jobs/kubernetes-sigs/tejolote/tejolote-presubmits.yaml:3]: ok will review some jobs

**Comment by cpanato** [comment]
PTAL @ameukam

**Comment by puerco** [comment]
Thank you @cpanato !!

**Pull request merged**